iconservice/icon_service_engine.py

```diff
@@ -1689,10 +1709,12 @@ def commit(self, _block_height: int, instant_block_hash: bytes, block_hash: Opti
             self._init_global_value_by_governance_score(context)
 
         if precommit_data.revision >= REV_IISS:
+            context.engine.iiss.send_calculate(context, precommit_data)
+
             context.engine.prep.commit(context, precommit_data)
             context.storage.rc.commit(precommit_data.rc_block_batch)
-            context.engine.iiss.send_ipc(context, precommit_data)
             # todo: consider case when error being raised in send ipc
+            context.engine.iiss.send_ipc(context)
```

Comment:
Need to check the reward calculator as a sequence of the message has been changed.

Reply:
I have checked this issue. RC does not consider the sequence of a commit, calculate.